### PR TITLE
Added ItaSA subtitle provider

### PR DESCRIPTION
**Pull request by Elettronik**
Proposed changes in this pull request:
- Updated subliminal library with code in the next release
- Added configurations for itasa subtitle provider

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: lib/subliminal/cli.py [hunk 1]
```diff
@@ -237,8 +237,8 @@ def subliminal(ctx, addic7ed, legendastv, opensubtitles, subscenter, cache_dir,
     ctx.obj = {'provider_configs': {}}
     if addic7ed:
         ctx.obj['provider_configs']['addic7ed'] = {'username': addic7ed[0], 'password': addic7ed[1]}
-    if legendastv:
```
Comment (miigotu) [inline lib/subliminal/cli.py:240]: this is removing the context for LTV

Comment (Elettronik) [inline lib/subliminal/cli.py:240]: Restored the cli configuration for LTV

**Comment by miigotu** [comment]
Now missing itaSA in cli call at https://github.com/SickRage/SickRage/pull/1119/files#diff-3980d21207b1478ad61c8156cd352813R216

**Comment by Elettronik** [comment]
@miigotu  amended

**Pull request merged**